src/components/Canopy.tsx

```diff
@@ -54,6 +56,10 @@ export const Canopy = function Canopy({ renderTarget }: CanopyViewProps) {
       gl.domElement.clientWidth,
       gl.domElement.clientHeight
     );
+
+    // This enables `#include <conjurer_common>`
+    ShaderChunk.conjurer_common = conjurerCommon;
```

Comment:
tbh, not sure if this is the best place to do this. But it seems to work, and I expect it's idempotent.

Reply:
Very cool, this will be such an improvement. Copy pasta helper functions was getting pretty old.